manim/animation/indication.py

```diff
@@ -342,7 +343,23 @@ def __init__(self, vmobject, n_segments=10, time_width=0.1, remover=True, **kwar
 #   Remove?
 #   Deprecate?
 #   Keep and add docs?
+@deprecated(reason="Use Create then FadeOut to achieve this effect.")
 class ShowCreationThenFadeOut(Succession):
+    """Creates mobject first, then fades out.
+
+    .. attention::
+       This animation method will be removed in the upcoming versions. Use Create and FadeOut separately.
+
+    Examples
+    --------
+    .. manim:: Create_FadeOut
+
+        class Create_FadeOut(Scene):
```

Comment:
Convention for classes is to use CamelCase for class names

```suggestion
    .. manim:: ShowCreationThenFadeOutExample

        class ShowCreationThenFadeOutExample(Scene):
```